pkg/apis/v1alpha5/provisioner_validation.go

```diff
@@ -305,3 +307,21 @@ func ValidateRequirement(requirement v1.NodeSelectorRequirement) error { //nolin
 	}
 	return errs
 }
+
+// Validate validateImageGCHighThresholdPercent
+func (kc *KubeletConfiguration) validateImageGCHighThresholdPercent() (errs *apis.FieldError) {
+	if kc.ImageGCHighThresholdPercent != nil && ptr.Int32Value(kc.ImageGCHighThresholdPercent) < ptr.Int32Value(kc.ImageGCLowThresholdPercent) {
```

Comment:
This is a nice sanity check but will kubelet actually complain if this is the case? If not, it might be better to just ignore this validation and let the user put themselves in this situation

Reply:
based on [kubernetes tests](https://github.com/kubernetes/kubernetes/blob/12c71fdf1cf96d756ff84382adf3764af0a76d57/pkg/kubelet/apis/config/validation/validation_test.go#L188-L195) it should fail if ImageGCHighThresholdPercent > ImageGCHighThresholdPercent.  I tough about implementing the same logic here.